Libplanet.Explorer/Queries/ExplorerQuery.cs

```diff
@@ -84,7 +84,7 @@ public ExplorerQuery(IBlockChainContext chainContext)
 
             foreach (var index in indexList)
             {
-                var block = store.GetBlock(index.value);
+                var block = store.GetBlock(index.value)!;
```

Comment:
We don't need `!` here. Fix lines `74` to `78` instead.

Reply:
Sorry, I misread as `store.GetBlock(index.value!)` instead of `store.GetBlock(index.value)!`. In that case, you can leave the lines 74-78 intact and throw an exception here I suppose.